src/frontend/sharedComponents/Button.tsx

```diff
@@ -15,60 +10,29 @@ import { TouchableNativeFeedback } from "./Touchables";
 
 type ColorScheme = "dark" | "light";
 type Variant = "contained" | "outlined" | "text";
-type Size = "medium" | "large";
-
-interface SharedTouchableProps {
-  disabled?: boolean | null;
-  onPress: (event: GestureResponderEvent) => void;
-}
 
 interface Props {
-  TouchableComponent?: React.ComponentType<Partial<SharedTouchableProps>>;
   children: React.ReactNode;
   color?: ColorScheme;
   disabled?: boolean;
   fullWidth?: boolean;
   onPress: (event: GestureResponderEvent) => void;
-  size?: Size;
   style?: ViewStyleProp;
   testID?: string;
   variant?: Variant;
 }
 
 const Button = ({
```

Comment:
don't think this component shouldn't have any changes (i'd just copy+paste whatever's in the shared branch to yours)